test/threadstest.c

```diff
@@ -437,7 +437,7 @@ static void thread_shared_evp_pkey(void)
             goto err;
 
         if (!TEST_int_ge(EVP_PKEY_decrypt_init(ctx), 0)
-                || !TEST_int_ge(EVP_PKEY_decrypt(ctx, ptbuf, &ptlen, ctbuf, ctlen),
+                || !TEST_int_gt(EVP_PKEY_decrypt(ctx, ptbuf, &ptlen, ctbuf, ctlen),
```

Comment:
ptlen needs to be reinitialized to sizeof(ptbuf) every time.

Reply:
@PeiweiHu can you please fix that?

Reply:
ok, sorry for being late. I will do it now.